docs/operations/export_api.md

```diff
@@ -0,0 +1,97 @@
+# Export API
+
+## Prerequesites
+None
```

Comment:
why None if the feature gate must be enabled? isnt it a Prerequesite?

Reply:
Because featuregates are a separate item in the documentation. So besides the featuregate, there are none.

Reply:
Yeah I think I'm with @ShellyKa13.  Seems like exactly the sort of thing that should go here.

Reply:
Well all the other pages have separate sections for enabling the feature gate. I think I am going to just remove the prerequesites section since it makes no sense.